roles/pulp3/defaults/main.yml

```diff
@@ -4,8 +4,8 @@ pulp_cache_dir: '/var/cache/pulp'
 pulp_config: {}
 pulp_config_dir: '/etc/pulp'
 pulp_default_admin_password: ''
-pulp_install_dir: '/opt/pulp'
-pulp_install_plugins: []
+pulp_install_dir: '/usr/local/lib/pulp'
+pulp_install_plugins: {}
```

Comment:
Why did we switch this to a dict again? It was something about storing extra data w.r.t each plugin. Is that extra data used anywhere? It may be easier if this was just a `[]` again. What do you think?

Reply:
Oh I know why:  `app_label: Used to make migrations.`

Reply:
1. Each plugin needs to specify the app label. So far, for each plugin the app label is "pulp_<plugin>", which is just "pulp-<plugin>".replace("-", "_"), but this is not enforced. The plugin writer can include whatever app label they want in their plugin config object. 
2. Each plugin accepts a "source_dir", which if set will install from source rather than PyPI. This allows a user to install some plugins from source, and others from PyPI.

Reply:
Oh word. Maybe add those to the commented example.